bindings/java/src/org/sleuthkit/datamodel/OsAccountRealmManager.java

```diff
@@ -812,4 +896,37 @@ void mergeRealms(OsAccountRealm sourceRealm, OsAccountRealm destRealm, CaseDbTra
 			db.releaseSingleUserCaseReadLock();
 		}
 	}
+	
+	/**
+	 * Status of a realm update.
+	 */
+	public enum RealmUpdateStatusEnum {
+
+		NO_CHANGE,	/// no change was made to account.
+		UPDATED,	/// account was updated
+		MERGED		/// account update triggered a merge
+	}
+	
+	/**
+	 * Container to encapsulate the status returned by the realm update api, and
+	 * the updated realm.
+	 */
+	public final static class RealmUpdateStatus {
+		
+		private final RealmUpdateStatusEnum updateStatusCode;
+		private final OsAccountRealm updatedRealm; // may be null if there is no update.
```

Comment:
I believe it has been agreed that an OsAccountRealm object will always be returned. Also, we should use the name changes: RealmUpdateStatus => OsRealmUpdateResult and RealmUpdateStatusEnum => OsRealmUpdateStatus.